sources/dfmc/reader/lexer.dylan

```diff
@@ -1032,7 +1035,7 @@ define method parse-integer
     else
       genericnegative(repeat(start, 0));
     end if;
-  exception (overflow :: <error>)
+  exception (overflow :: <arithmetic-error>)
```

Comment:
Should this be part of a separate commit? (And the next couple of changes as well...)

Reply:
Yes, good point. I'll clean it up in several places.

Reply:
I'm just dropping this change, actually, since I can't remember what it was for, exactly, and I can't guarantee `<arithmetic-error>` was the only kind of error it was intended to catch.